hw/top_earlgrey/ip/alert_handler/data/autogen/alert_handler.hjson

```diff
@@ -48,7 +48,7 @@
     { name: "NAlerts",
       desc: "Number of peripheral inputs",
```

Comment:
The desc field here and below seems goofy.

Reply:
yeah this seems to be out of date. will fix, thanks!